src/deploy/signal.tf

```diff
@@ -0,0 +1,82 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+resource "aws_instance" "signal" {
+  ami = "ami-0a313d6098716f372"
+  instance_type = "t2.micro"
+  key_name = "deployer-key"
+
+  # Copies scripts
```

Comment:
This is the simplest code deployment. It assumes you will deploy a new instance at the same time as a code update. In a more complex situation, I'd use the Terraform AWS CodeDeploy plugin, Hashicorp's Nomad, or one of the in-instance configuration management tools like Chef.